coordinatorlib/src/it/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMapWriterIndexDirectiveTest.scala

```diff
@@ -14,7 +14,7 @@ trait PostgresDatasetMapWriterIndexDirectiveTest { this: PostgresDatasetMapWrite
       val directiveEnabled = JObject(Map("enabled" -> JBoolean.canonicalTrue))
       tables.createOrUpdateIndexDirective(ci1, directiveEnabled)
 
-      val indexes  = tables.indexDirectives(vi1)
```

Comment:
can we add a test that does use the new "just for this column" functionality?

Reply:
added